vignettes/agg_scale_performance.Rmd

```diff
@@ -0,0 +1,175 @@
+---
+title: "Aggregation/Scaling performance"
+output: rmarkdown::html_vignette
```

Comment:
Do you have a knit html file saved somewhere that I can open? I think it's useful on PRs to include knit vignettes so that reviewers don't have to go knit themselves.